docs/fault-proof-alpha/immunifi.md

```diff
@@ -0,0 +1,82 @@
+# Fault Proof Alpha Bounty
+
+The fault proof alpha system will be deployed to the Goerli testnet as a sidecar to the current system. During the alpha, the outcome of fault dispute games will have no influence on the official bridge contracts nor the official `L2OutputOracle`.
+During this early phase of ongoing development, we invite security researchers and developers to engage with the system and attempt to break its current components.
+
+The current system is not production ready, however the core infrastructure for creating an instruction trace ([Cannon][cannon] + the [`op-program`][op-program]), the off-chain challenge agent ([`op-challenger`][op-challenger]),
+and the on-chain infrastructure for the [Dispute Game][dispute-game] are all in place.
+
+## Known Issues
+The alpha system is not prepared for mainnet, and as such, there are a number of known issues that we are working on fixing and components of the system that must be improved prior to it being sustainable.
+
+1. DoS attacks are currently likely to occur due to the lack of bonds in the alpha system as well as the lack of an extra layer of bisection in the dispute game to reduce the running time of [Cannon][cannon]. It is possible to
+    DoS the network of honest challengers by creating a large number of invalid challenges.
+
+### Reviewer Notes
+1. **Any bug report without a PoC in the form of a test in `op-e2e` will not be considered a valid bug report.**
+    1. *todo*: Provide Adrian's example op-e2e test
```

Comment:
Can link to https://github.com/ethereum-optimism/optimism/blob/develop/docs/fault-proof-alpha/invalid-proposals.md now